appinventor/buildserver/src/com/google/appinventor/buildserver/AabZipper.java

```diff
@@ -0,0 +1,65 @@
+package com.google.appinventor.buildserver;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class AabZipper {
```

Comment:
Add a short Javadoc explaining what this class does. Since it looks like this is a utility class, you may want to create a private constructor so it cannot be instantiated and move it into the `util` package.